website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+  Packer can be configured to publish artifact metadata for all completed builds to an active HCP Packer registry.
+  The configuration for HCP Packer is managed by environment variables for legacy JSON templates or via a hcp_packer_registry
+  block for HCL2 templates. 
+page_title: Configuring Packer for HCP Packer
+---
+
+# Configuring Packer for HCP Packer
+
+HCP Packer stores metadata about your Packer images so that you can track updates, use the most up-to-date base images, and deploy the most up-to-date downstream images. 
+For more details on How it Works see the [HashiCorp Cloud Platform Packer documentation](https://cloud.hashicorp.com/docs/packer#what-is-hcp-packer) 
+
+You can configure any Packer template to push metadata to the HCP Packer registry, but we recommend that you use HCL2 Packer templates. 
+The HCL2 support in Packer allows you to codify how Packer interacts with the registry and customize your image metadata. 
+
+Users wishing to use HCP Packer with JSON are encouraged to get started with the environment variables and eventually work to migrate from JSON to HCL2 configuration
+templates. 
```

Comment:
```suggestion
[HCP Packer](https://cloud.hashicorp.com/docs/packer) stores metadata about your Packer images so that you can track updates, use the most up-to-date base images, and deploy the most up-to-date downstream images. 

You can configure any Packer template to push metadata to the HCP Packer registry, but we recommend that you use HCL2 Packer templates. The HCL2 support in Packer allows you to codify how Packer interacts with the registry and customize your image metadata. Refer to [Template Configuration](https://cloud.hashicorp.com/docs/packer/store-image-metadata/template-configuration) in the HCP Packer documentation for details. 

If you want to use HCP Packer with JSON, we recommend first connecting with [HCP Packer environment variables](#hcp-packer-environment-variables) and then migrating from JSON to HCL2 configuration templates. 
```